modules/nextflow/src/main/groovy/nextflow/processor/TaskArrayCollector.groovy

```diff
@@ -56,6 +55,9 @@ class TaskArrayCollector {
             // only needed for container-native executors and/or Fusion
             'container',
             'containerOptions',
+            // only needed when using Wave
+            'conda',
+            'spack',
```

Comment:
Wave is not supporting anymore `spack`. Do you think it needed anycase?

Reply:
oh, in that case it should not be needed. I will revert